config/config.go

```diff
@@ -518,7 +515,15 @@ func (c WorkerSettings) MaxWorkerCount() int {
 
 func (c WorkerSettings) WorkerRollingUpdateMinInstancesInService() int {
 	if c.AutoScalingGroup.RollingUpdateMinInstancesInService == 0 {
-		return c.MaxWorkerCount() - 1
+
+		maxWorkerCount := c.MaxWorkerCount()
+
+		//never accept worker size = 0
+		if maxWorkerCount > 0 {
+			return maxWorkerCount - 1
+		}
+
+		return 1
```

Comment:
Any reason you don't want to return 0 instead of 1 as the minimum value?
I guess returning 1 here would make it impossible to set size of the ASG to zero, which is currently possible and even possible after #315?